lib/node_modules/@stdlib/stats/base/dists/frechet/kurtosis/README.md

```diff
@@ -158,11 +158,104 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- Section to include cited references. If references are included, add a horizontal rule *before* the section. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
 
-<section class="references">
```

Comment:
If you remove this section, the comment in the preceding line needs to be removed as well.